### DISCUSSION TITLE
Comparison operator looks different

### DISCUSSION THREAD

**Discussion by edwardjjj** [LazyVim/LazyVim#2105] [Q&A] [answered]
## Comparison Operator
Just noticed the comparison operators looks like this:
![image](https://github.com/LazyVim/LazyVim/assets/76624665/37ac9c04-002e-4296-979e-3a545118dd04)
Can someone help me locate which config to change the behavior back to !=, >=, <=, etc. Thank you very much!

**Comment by dpetka2001** [comment, marked as answer]
This depends on the font you're using.

**Reply by edwardjjj** [reply]
Thank you very much for your information. Turns out it is the JetBrains mono font.

**Reply by dpetka2001** [reply]
Some terminals also support an option to not show ligatures. You could just disable ligatures, if you'd like to keep the font you're using.
